CONTRIBUTING.md

```diff
@@ -503,4 +503,4 @@ We are running a Discord server. You can go here to talk, discuss, and more with
 | Visual Studio Code | https://code.visualstudio.com/                     | A code editor for editing code                     |
 | vscode.dev         | https://vscode.dev/                                | Visual Studio Code in the browser                  |
 | Iloveimg           | https://www.iloveimg.com/resize-image/resize-svg   | Resizing SVG's                                     |
-| svgviewer.dev      | https://www.svgviewer.dev/                         | View, save, and optimize SVGs                      |
\ No newline at end of file
+| svgviewer.dev      | https://www.svgviewer.dev/                         | View, save, and optimize SVGs                      |
```

Comment:
I don't see a change here. Could you point out what exactly changed on this line?

Reply:
I'm not sure about that, but I didn't make any changes there.

Reply:
it's maybe the line ending / line break, check https://stackoverflow.com/questions/1552749/difference-between-cr-lf-lf-and-cr-line-break-types

Reply:
I'll approve once this diff is fixed. Invisible changes always leave me with a bad taste in the mouth